test/sbvr/pilots.js

```diff
@@ -169,7 +169,7 @@ SELECT NOT EXISTS (
 SELECT NOT EXISTS (
 	SELECT 1
 	FROM "pilot" AS "pilot.0"
-	WHERE "pilot.0"."is experienced" = 1
+	WHERE "pilot.0"."is experienced" = TRUE
```

Comment:
I guess changing to real booleans allows us to do an extra optimization for PG in a follow-up minor PR that will convert this to
```
WHERE "pilot.0"."is experienced"
```
right? Extra 4 bytes sql statement size savings :tada:

Reply:
Yeah that should be possible in future